### PR TITLE
add reset button and share button to budget function

### PR DESCRIPTION
**Pull request by xjackk**
* https://federal-spending-transparency.atlassian.net/browse/DA-5113

Added Reset Button and Share to `Budget Function` page

### CODE REVIEW

File: src/components/visualizations/sankey/sankey.jsx [hunk 1]
```diff
@@ -430,24 +437,22 @@ function Sankey(props) {
...
-      .text('The quick brown fox jumped over the lazy dog 1.');
-
-    d3.select("#sankey-viz svg")
-      .append('desc')
-      .attr('id', 'descTest')
-      .text('The quick brown fox jumped over the lazy dog 1.  The quick brown fox jumped over the lazy dog 2.  ' +
-        'The quick brown fox jumped over the lazy dog 3.  The quick brown fox jumped over the lazy dog 4. ' +
-        'The quick brown fox jumped over the lazy dog 5.');
-
+    // d3.select("#sankey-viz svg")
```
Comment (matthewbrown314) [inline src/components/visualizations/sankey/sankey.jsx]: Good catch. The real problem here is that dummy text was used to fulfill the title and desc requirements for 508. Could you @ Kelli in dl-gatsby and say that the viz in budget-function had dummy 508 text for its title and description and likely passed Ritesh's 508 scans as a result?

**Comment by matthewbrown314** [comment]
Everything looks good; however, the reset button needs to also reset the radio button selection back to FY19.

File: src/components/visualizations/sankey/sankey.jsx (hunk 1 above)
Comment (xjackk) [reply src/components/visualizations/sankey/sankey.jsx]: Yes, can do

**Comment by matthewbrown314** [comment]
> Everything looks good; however, the reset button needs to also reset the radio button selection back to FY19.

Unfortunately, the radio button updates to fy19; but, nothing else within the Viz resets back to the original. This is especially apparent when going between fy17 and fy19 as fy17 has "Grants, Subsidies, and Contributions" as the 2nd block on the right, while fy19 has it as the 3rd. The dollar amounts also are reflective of the previously selected option.

* Be aware that in the current code setup, if you have fy17 (or fy18) selected and hit reset, that clicking between that same radio button and fy19 will not update the viz for some reason. You have to click the third radio button to finally get the viz to update appropriately.

**Comment by matthewbrown314** [comment]
Looks great!

**Pull request merged**